.changeset/sweet-pants-tell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/inputs': major
```

Comment:
Should be `minor` as it does not include breaking changes, but new features.

Reply:
Shit. Should have spotted that.

Reply:
I thought that since a new package is added to the presets, it should be a major change for presets package. But changeset would consider that a major change for the whole repo. Sorry for the confusion